WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -993,16 +993,16 @@ def _writePilotScript( self, workingDirectory, pilotOptions, proxy = None,
   os.environ["X509_CERT_DIR"]=os.path.join(pilotWorkingDirectory, 'etc/grid-security/certificates')
   # TODO: structure the output
   print '==========================================================='
-  logger.debug('Environment of execution host\n')
+  logger.debug('Environment of execution host\\n')
   for key, val in os.environ.iteritems():
     logger.debug( key + '=' + val )
-  print '===========================================================\n'
+  print '===========================================================\\n'
 except Exception as x:
   print >> sys.stderr, x
   shutil.rmtree( pilotWorkingDirectory )
   sys.exit(-1)
 cmd = "python %(pilotScript)s %(pilotOptions)s"
-logger.info('Executing: %s' % cmd)
+logger.info('Executing: %%s' %% cmd)
```

Comment:
Is this really correct?

Reply:
I ran a couple pilots in the LHCb certification setup with this code, and worked OK.